api/GPUSupportedFeatures.json

```diff
@@ -212,7 +212,10 @@
                 "Available on all adapters and enabled automatically on all devices even if not requested."
               ]
             },
-            "chrome_android": "mirror",
+            "chrome_android": {
+              "version_added": "139",
+              "notes": "Available on all adapters and enabled automatically on all devices even if not requested."
```

Comment:
I understand that this is note is pre-existing, but what does it mean for web developers? Are there certain things that don't work, and why does it not justify `partial_implementation: true`?